### PR TITLE
Adding up changes related to Docker docker conmpose run and bugfix fo…

### PR DESCRIPTION
**Pull request by siddshadab**
…r runReport

### CODE REVIEW

File: birt-engine-autoconfig/src/main/java/com/innoventsolutions/birt/controller/RunController.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ public ResponseEntity<StreamingResponseBody> getTestHTML(final HttpServletRespon
 	 * StreamingResponseBody
 	 *
 	 */
-	@PostMapping(value = "/runReport")
+	@PostMapping(value = "/runReport",consumes = {"text/plain", "application/*"},produces = {"text/plain", "application/*"})
```
Comment (antho686) [inline birt-engine-autoconfig/src/main/java/com/innoventsolutions/birt/controller/RunController.java]: Add 1 space after each coma (just auto-format file)

File: birt-simple/run.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```
Comment (antho686) [inline birt-simple/run.sh:1]: So thoughts for later: add a parsing feature in the run file to find the latest version. duplicate the run file to get a run.sh, run.dev.sh and a run.local.sh. (I'll take notes and add this to the Trello if this solution is the one we end up going with).

Comment (antho686) [reply birt-simple/run.sh:1]: (you can mark this message as resolved)

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+version: '3.1'
+
+services:
+
+  birtapp:
+    image: birtapp/birt_engine
```
Comment (antho686) [inline docker-compose.yml]: rename the image name prefix to verseData/birt_engine

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+version: '3.1'
+
+services:
+
+  birtapp:
```
Comment (antho686) [inline docker-compose.yml]: rename the container's name to birtengine

**Review by antho686 — CHANGES_REQUESTED** [review]

File: birt-simple/run.sh (hunk 2 above)
Comment (siddshadab) [reply birt-simple/run.sh:1]: Is this still required or we can leave it

Comment (siddshadab) [reply birt-simple/run.sh:1]: because for now making this run script is just a one or two liner

Comment (antho686) [reply birt-simple/run.sh:1]: yeah, we'll keep this part for later when we'll integrate every piece of software under 1 installer

**Review by antho686 — APPROVED** [review]

**Comment by antho686** [comment]
you can squash and merge. for a commit message can you use something like: "BugFix - Docker Compose + API endpoints fix" or something similar. the key here is to have the prefix "Bugfix -" followed by a really short description of the work done.

**Comment by siddshadab** [comment]
Sure will keep in mind before pushing something

**Comment by antho686** [comment]
> 

You can push whatevere you want whenever you want, the key is just to merge only clean short and concise commits to the main branch. After a year or two, it'll be much easier to navigate thru the changes haha

**Pull request merged**